blog.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="zxx">
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>Adopt A Dog</title>
+      <link rel="stylesheet" href="stylesheet/style.css">
+   </head>
+   <body>
+      <header>
+         <div class="header-container">
+            <div class="logo-bar">
+               <img src="images/adoptadog-logo.svg" alt="">
+            </div>
+            <div class="navigation-bar">
+               <nav>
+                  <ul>
+                     <li><a href="index.html">Home / </a></li>
+                     <li> <a href="dogs.html">Dogs / </a></li>
+                     <li><a href="blog.html">Blog </a></li>
+                  </ul>
+               </nav>
+            </div>
+         </div>
+      </header>
+      <div><h2>Adoptadog Blog</h2></div>
+      <div class="blog-container">
+         <div class="blog-container-box blog-image-container-box">
+            <img src="images/blog-1.jpg" alt="">
+         </div>
+         <div class="blog-container-box">
+            <h3>Traveling With Your Dog</h3>
+            Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.
+            Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit
+            ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam
+            sus am aut amet ant fugiatum, utem non reptat.
+            Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur,
+            con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+            qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim
+            quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.
+         </div>
+      </div>
+      <div class="blog-container 2">
+         <div class="blog-container-box">
```

Comment:
You shouldn't need a div around an image. Keep the divs to a minimum, and generally they are used to group multiple elements together.